cognite_toolkit/_cdf_tk/data_classes/_packages.py

```diff
@@ -66,7 +66,7 @@ def load(
             root_module_dir: The module directories to load the packages from.
         """
 
-        package_definition_path = root_module_dir / "package.toml"
+        package_definition_path = next(root_module_dir.rglob("packages.toml"))
```

Comment:
Needs an alpha flag I think, or is `packages.toml` only in the builtint?

What happens if you don't find it?